modules/menu/menu.c

```diff
@@ -544,6 +545,40 @@ static int hold_prev_call(struct re_printf *pf, void *unused)
 	return call_hold(ua_prev_call(uag_cur()), true);
 }
 
+static int switch_audio_dev(struct re_printf *pf, void *arg)
+{
+	const struct cmd_arg *carg = arg;
+	static bool switch_aud_inprogress;
+	char *mod;
+	char *device;
+	int err = 0;
+
+	if (!switch_aud_inprogress && !carg->complete) {
+		statmode = STATMODE_OFF;
+		re_hprintf(pf, "\rPlease enter transfer target SIP uri:\n");
```

Comment:
the text does not match the expected user input, it should be something
like: "Please enter new audio device in format driver,device"